icu4c/source/test/testdata/Makefile.in

```diff
@@ -92,6 +92,7 @@ TESTSRCDATADIR=$(top_srcdir)/test/testdata
 TESTOUTDIR=$(top_builddir)/test/testdata/out
 BUILD_DIRS = $(TESTOUTDIR) $(TESTBUILDDIR) $(TESTOUTDIR)/$(TESTDT)
 GENTEST=$(TOOLDIR)/gentest/gentest$(TOOLEXEEXT)
+LIBRARY_DATA_DIR=$(OUTDIR)/build
 
 ifeq ($(PKGDATA_MODE),common)
 ICU_DATA_OPT = -i $(OUTDIR)
```

Comment:
I just noticed that there is already this `ICU_DATA_OPT` being set, but I don't know if it's being used anywhere. Hmm.